src/shared/extensionUtilities.ts

```diff
@@ -202,3 +203,33 @@ export function toastNewUser(context: vscode.ExtensionContext): void {
         getLogger().error(err as Error)
     }
 }
+
+/**
+ * Creates a modal to display OS, AWS Toolkit, and VS Code
+ * versions and allows user to copy to clipboard
+ * Also prints to the toolkit output channel
+ *
+ * @param toolkitOutputChannel VS Code Output Channel
+ */
+
```

Comment:
unnecessary empty line

Reply:
Thanks, I've taken that out in the last commit.